src/extension.ts

```diff
@@ -235,6 +198,7 @@ function initAsyncCommand(context: vscode.ExtensionContext, commandId: string, c
             } else {
                 result = 'Failed';
                 errorData = util.errToString(err);
+                vscode.window.showErrorMessage(errorData);
```

Comment:
errorData returns a stringified object including 'Error' and 'Message' fields. We just want the message part to be displayed